test/avro_schema_test.cc

```diff
@@ -1070,3 +1075,135 @@ TEST(AvroSchemaProjectionTest, ProjectDecimalIncompatible) {
 }
 
 }  // namespace iceberg::avro
+
+// NameMapping tests for Avro schema context
+namespace iceberg::avro {
+
+std::unique_ptr<NameMapping> CreateTestNameMapping() {
```

Comment:
Could you please revert changes in this file? Or we can move them to `name_mapping_test.cc`.